src/harness/certs/README.md

```diff
@@ -24,6 +24,13 @@ https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/
                    sas_expired                                               domain_proxy_expired
                                                                              domain_proxy_inapplicable
 
+                  -------------root_ca --------------------------------------                
+                 /                   \         				      \ 
+                /                     \         			       \
+       blacklisted_sas_ca         blacklisted_cbsd_ca   		blacklisted_proxy_ca
```

Comment:
Probably `revoked_sas_ca` is more accurate (similar for the others).

Reply:
sub CA's   are named correctly to give better clarity